### PR TITLE
Add Time scalar

### PR DESCRIPTION
**Pull request by gdalmau**
Add `Time` scalar along with the existing `Date` and `DateTime`

PR for the docs: https://github.com/mirumee/ariadne-website/pull/66

### CODE REVIEW

**Comment by gdalmau** [comment]
The Pull Request is ready to be reviewed btw 😄

**Comment by rafalp** [comment]
@gdalmau apologies. I remember about those Django PRs, but I am constantly finding myself short on time to do so :(

But I am considering those must haves for next Ariadne release, as per the roadmap for next release to be last one to provide Django utilities as part of main package.

File: tests/django/test_scalars.py [hunk 1]
```diff
@@ -5,10 +5,13 @@
 from ariadne.contrib.django.scalars import (
     date_scalar,
     datetime_scalar,
+    time_scalar,
```
Comment (rafalp) [inline tests/django/test_scalars.py:8]: Please sort imports alphabetically - `time_scalar` should go under `serialize_time`.

**Review by rafalp — CHANGES_REQUESTED** [review]

**Comment by rafalp** [comment]
Only single nitpick, but once this is fixed I'll be happy to merge this. Thanks!

File: tests/django/test_scalars.py (hunk 1 above)
Comment (gdalmau) [reply tests/django/test_scalars.py]: Done! Maybe it would be good to make travis run `isort`?

**Review by rafalp — APPROVED** [review]

**Pull request merged**